src/main/java/org/folio/edge/oaipmh/OaiPmhHandler.java

```diff
@@ -251,4 +283,45 @@ private void handleNotAcceptableError(RoutingContext ctx, HttpServerRequest requ
     notAcceptable(ctx,
         "Accept header must be \"text/xml\" for this request, but it is " + "\"" + unsupportedType + "\"" + ", can not send */*");
   }
+
+  /**
+   * This method make request to mod-configuration module and receive config setting associate with name behavior
+   *
+   * @param ctx routing context
+   * @return value of field errorsProcessing
+   */
+  private CompletableFuture<String> getErrorsProcessingConfigSetting(RoutingContext ctx) {
+    String QUERY = "module==OAIPMH and configName==behavior";
```

Comment:
It would be better to move it to contsants or build query here by using CQLQueryBuilder.java

Reply:
I used constant